diff --git a/src/QF_API.ml b/src/QF_API.ml
index 45076f1..e794f10 100644
--- a/src/QF_API.ml
+++ b/src/QF_API.ml
@@ -41,7 +41,7 @@ let generalize_model model ~true_of_model ~rigid_vars ~newvars =
       let terms = HTerms.find tbl value in (* list of rigid variables that have that value *)
       let value =
         match Term.reveal value with
-        | Term App(f, [arg]) when Term.equal f (Model.epsilon_real()) ->
+        (*| Term App(f, [arg]) when Term.equal f (Model.epsilon_real()) ->
            begin
              let Term arg = Term.reveal arg in
              match arg with
@@ -50,7 +50,7 @@ let generalize_model model ~true_of_model ~rigid_vars ~newvars =
                 let epsilon = Term.subst_term [yvar, var] body in
                 WithEpsilons.{ main; epsilons = [epsilon] }
              | _ -> failwith "should not be"
-           end
+           end*)
         | _ -> WithEpsilons.return value
       in
       print 3 "@[<v2>Trying to eliminate variable %a, with value %a and matching variables %a@]@,"
